### PR TITLE
Fix leaking file descriptor

### PR DESCRIPTION
**Pull request by tmattio**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ ENV OCAMLORG_REPO_PATH /var/opam-repository/
 ENV OCAMLORG_PKG_STATE_PATH /var/package.state
 ENV OCAMLORG_TOPLEVELS_PATH /var/toplevels/
 ENV OCAMLORG_DEBUG false
+ENV DREAM_VERBOSITY info
```
Comment (TheLortex) [inline Dockerfile:33]: Is that compatible with GDPR as long as we remove the logs once in a while ?

**Review by TheLortex — APPROVED** [review]
Thanks, this should fix the main issue.

For the logging, I believe we will obtain source IPs so we might need to be careful about that.

File: Dockerfile (hunk 1 above)
Comment (tmattio) [reply Dockerfile:33]: I don't see why it wouldn't be compliant: there's no personal data associated with the info logs, only the requested URLs.

I'm not a GDPR expert though 🤷

Comment (TheLortex) [reply Dockerfile:33]: Oh my bad, I thought the IP would be shown if logging requests was enabled. So it's ok we don't have any personal data.

**Pull request merged**